libvast/test/arrow_table_slice.cpp

```diff
@@ -83,7 +83,7 @@ TEST(manual table slice building) {
   // Our schema is `map: list<key: int64, value: int64>, int: int64`.
   auto kvp_fields = std::vector{field("key", int64()), field("value", int64())};
   auto kvp_record = struct_(kvp_fields);
-  auto map_record = list(kvp_record);
+  auto map_record = arrow::list(kvp_record);
```

Comment:
Can you remove the `using namespace arrow` from this scope entirely while you're at it?